.github/workflows/cleanup.yml

```diff
@@ -1,42 +0,0 @@
-name: cleanup
```

Comment:
Is this delete intentional ?

Reply:
this was intentional, but we can leave it out in the real PR